src/Order.java

```diff
@@ -1,33 +1,99 @@
 
-import java.util.Date;
+import java.awt.Choice;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.TreeSet;
 
 public class Order {
+	// class level variables
 	private int orderId;
-	private Date orderDate = new Date();
-	private Item[] orderCart = new Item[] {};
-	Order(int id, Date dateTime, Item[] item) {
-		this.orderId = id;
-		this.orderDate = dateTime;
-		this.orderCart = item;
-	}
+	private int customerId;
+	private LocalDateTime orderDate = LocalDateTime.now();
+	private TreeSet<Item> orderCart;
+	private static int counter;
+	private static HashMap<Integer, Bill> currentSale = new HashMap(); 
+	
+	// Getter-Setters
 	public int getOrderId() {
 		return orderId;
 	}
 	public void setOrderId(int orderId) {
 		this.orderId = orderId;
 	}
-	public Date getOrderDate() {
+	public int getCustomerId() {
+		return customerId;
+	}
+	public void setCustomerId(int customerId) {
+		this.customerId = customerId;
+	}
+	public LocalDateTime getOrderDate() {
 		return orderDate;
 	}
-	public void setOrderDate(Date orderDate) {
+	public void setOrderDate(LocalDateTime orderDate) {
 		this.orderDate = orderDate;
 	}
-	public Item[] getOrderCart() {
+	public TreeSet<Item> getOrderCart() {
 		return orderCart;
 	}
-	public void setOrderCart(Item[] orderCart) {
+	public void setOrderCart(TreeSet<Item> orderCart) {
 		this.orderCart = orderCart;
 	}
 	
-	// use this for getting date - LocalDateTime.now()
+	// Parameterized Constructor
+	Order(int id, LocalDateTime dateTime, TreeSet<Item> ss) {
+		this.orderId = id;
+		this.orderDate = dateTime;
+		this.orderCart = ss;
+	}
+	
+	@Override
+	public String toString() {
+		Iterator<Item> i = orderCart.iterator();
+		String displayableOrderCart = null;
+		Double totalCartPrice = 0.00;
+		while (i.hasNext()) {
+			Item item = (Item) i.next();
+			if(displayableOrderCart == null) {
+				displayableOrderCart = item.getItemName();
+			} else {
+				displayableOrderCart = displayableOrderCart.concat(item.getItemName());
+			}
+			totalCartPrice += item.getItemRate();
+			if(i.hasNext()) {
+				displayableOrderCart = displayableOrderCart.concat(", ");
+			}
+		}
+		String formattedOrderDate = ""+orderDate.getDayOfMonth()+"-"+orderDate.getMonth()+"-"+orderDate.getYear();
+		Bill billObj = new Bill(this.orderId, 456, totalCartPrice, orderDate);
+		currentSale.put(this.orderId, billObj);
+		return "Order Date: " + formattedOrderDate + ", Order Cart: " + displayableOrderCart + ", Total Price: "+ totalCartPrice;
+	}
+	public static void createOrder() {
+		int orderID = (int) Math.random();
+		LocalDateTime currentDateTime = LocalDateTime.now();
+		Scanner sc= new Scanner(System.in);
+		String choice = "n";
+		TreeSet<Item> ss = new TreeSet<Item>(new ItemListComparator());
```

Comment:
The Comparator used here is totally wrong

Reply:
Implement comparable in Item class

Reply:
Replace the type specification in this constructor call with the diamond operator ("<>").Use proper  meaningfull variable names

```suggestion
		TreeSet<Item> itemSet= new TreeSet<>();
```